common/service/dynamicconfig/constants.go

```diff
@@ -70,13 +70,13 @@ var keys = map[Key]string{
 	EnableNamespaceNotActiveAutoForwarding: "system.enableNamespaceNotActiveAutoForwarding",
 	TransactionSizeLimit:                   "system.transactionSizeLimit",
 	MinRetentionDays:                       "system.minRetentionDays",
-	MaxWorkflowTaskTimeout:                 "system.maxWorkflowTaskTimeout",
```

Comment:
I think we should keep this dynamic config knob for operations purpose.

Reply:
talked offline and put here for visibility purpose

there is a per namespace dynamic config for workflow task, user can use that dynamic config to override the timeout.

this PR add more strict upper limit (non overridable) to the workflow task timeout, e.g. task timeout (even with dynamic config) cannot exceeds this strict upper limit (2 minutes)